docs/bugs.json

```diff
@@ -1,4 +1,16 @@
 [
+    {
+        "uid": "SOL-2022-6",
+        "name": "ABIEncoderV2HeadOverflowWithStaticArrayCleanup",
```

Comment:
Not actually sure, we'd have needed to emphasize "ABIEncoderV2" that much, given that it's long been the default now. But also fine.